Tests/test_QFT_value_002.aplf

```diff
@@ -0,0 +1,12 @@
+test_QFT_value_002 ← {
+    ⍝ Testing QFT for up to 6 qubits
+    ⍝ Create all basis states for up to 6 qubits
+    input ← {(((,∘1)(⍴⊢))⍴⊢)¨↓↑1,⍨¨0/⍨¨1-⍨⍳⍵}¨2*⍳6
```

Comment:
Very, very useful! Can you add this as a new function in `#.quapl.mlt`?

Reply:
Just added it! Let me know if it looks okay